nodes/node.go

```diff
@@ -6,6 +6,9 @@ import (
 
 // Interface for defining unit of work to be processed by event loop
 type InterfaceNode interface {
+	GetName() string
+	SetName(name string)
+	InitializeFields()
  	ProcessVirt(predecessorNodeResults []interface{})
```

Comment:
Rename `predecessorNodeResults` to `predecessorNodesResults`

Reply:
makes sense - done